javascript/javascript_sdk/src/message_receiver.js

```diff
@@ -1,57 +1,57 @@
 'use strict';
 
-const zmq = require('zeromq');
-
+const {OutputServiceClient} = require('./../js_proto/output_grpc_web_pb.js');
 const parsing = require('./parse_output');
 
 
 class MessageReceiver {
   constructor() {
-    this._socket = new zmq.Subscriber;
+    //this._socket = new zmq.Subscriber;
```

Comment:
Let's just remove the comment since we're not using zmq any longer. Can we remove all the comments in the file?

Reply:
Sorry, all the code under javascript will be retreat in the other PR. Sorry for the inconvenience.